code/modules/mob/living/say.dm

```diff
@@ -40,18 +40,18 @@ var/list/department_radio_keys = list(
 
 	  //kinda localization -- rastaf0
 	  //same keys as above, but on russian keyboard layout. This file uses cp1251 as encoding.
-	  ":�" = "right ear",	".�" = "right ear",

```

Comment:
you nuked the file encoding, need to go back in vscode, select encoding in the bottom right
![https://i.tigercat2000.net/2021/05/O5DGpfTUEw.png](https://i.tigercat2000.net/2021/05/O5DGpfTUEw.png)
and set it to ![https://i.tigercat2000.net/2021/05/7HCB0QYzt0.png](https://i.tigercat2000.net/2021/05/7HCB0QYzt0.png)

Reply:
The other ones below don't super matter except for visibility, this one **must** be fixed or it breaks shit

Reply:
The eye is a UTF character, though.

Reply:
Nevermind I read this wrong, the eye is in another file. STILL. I'm gonna convert that to UTF because EH.